Resources/Locale/en-US/health-examinable/health-examinable-carbon.ftl

```diff
@@ -15,4 +15,8 @@ health-examinable-carbon-Heat-25 = [color=orange]{ CAPITALIZE(SUBJECT($target))
 health-examinable-carbon-Heat-50 = [color=orange]{ CAPITALIZE(SUBJECT($target)) } { CONJUGATE-HAVE($target) } major burns across { POSS-ADJ($target) } body.[/color]
 health-examinable-carbon-Heat-75 = [color=orange]{ CAPITALIZE(SUBJECT($target)) } { CONJUGATE-HAVE($target) } severe third-degree burns across { POSS-ADJ($target) } body![/color]
 
+health-examinable-carbon-Cold-25 = [color=lightblue]{ CAPITALIZE(SUBJECT($target)) } { CONJUGATE-HAVE($target) } minor frostbite across { POSS-ADJ($target) } body.[/color]
+health-examinable-carbon-Cold-50 = [color=lightblue]{ CAPITALIZE(SUBJECT($target)) } { CONJUGATE-HAVE($target) } major frostbite across { POSS-ADJ($target) } body.[/color]
+health-examinable-carbon-Cold-75 = [color=lightblue]{ CAPITALIZE(SUBJECT($target)) } { CONJUGATE-HAVE($target) } severe third-degree frostbite across { POSS-ADJ($target) } body![/color]
```

Comment:
frostbite is cell rupturing from ice crystals or something i dont think third degree applies

Reply:
"Severity may be divided into superficial (1st and 2nd degree) or deep (3rd and 4th degree)"
https://en.m.wikipedia.org/wiki/Frostbite